benchmarks/pathway.js

```diff
@@ -0,0 +1,21 @@
+var Graph  = require('../dist/src/Graph')
```

Comment:
Do you already have a script or command that runs these benchmarks, or are they not for public consumption?